### PR TITLE
feat: page with sound vibrations

### PR DESCRIPTION
**Pull request by juliahermak**
Screenshot 2024-01-05 21 52 37

### CODE REVIEW

**Comment by juliahermak** [comment]
[Feat: Vibro](https://trello.com/c/iudwekjv/406-feat-vibro)

File: src/lib/vibrate.js [hunk 1]
```diff
@@ -1,10 +1,11 @@
...
-  MEDIUM: [80],
+  VERY_SHORT: [40],
+  SHORT: [80],
+  MEDIUM: [160],
   LONG: [300],
-  DOUBLE_VERY_SHORT: [25, 25],
-  TRIPLE_VERY_SHORT: [25, 25, 25]
+  DOUBLE_VERY_SHORT: [40, 20, 40],
+  TRIPLE_VERY_SHORT: [40, 20, 40, 20, 40],
+  DOUBLE_SHORT: [80, 40, 80]
```
Comment (adamant-al) [inline src/lib/vibrate.js:8]:
```suggestion
  DOUBLE_VERY_SHORT: [40, 40, 40],
  TRIPLE_VERY_SHORT: [40, 40, 40, 40, 40],
  DOUBLE_SHORT: [80, 80, 80]
```

**Review by adamant-al — CHANGES_REQUESTED** [review]
Short pauses don't feel like pauses.

File: src/lib/vibrate.js (hunk 1 above)
Comment (adamant-al) [reply src/lib/vibrate.js:8]: 60 is the most suitable pause

**Comment by adamant-al** [comment]
I'm OK to merge it.

**Review by bludnic — APPROVED** [review]

**Pull request merged**